packages/broker/configs/docker-1.env.json

```diff
@@ -24,10 +24,10 @@
   "streamrUrl": "http://10.200.10.1",
   "streamrAddress": "0xFCAd0B19bB29D4674531d6f115237E16AfCE377c",
   "storageNodeConfig": {
-    "registry": {
```

Comment:
The old config seemed correct here. Why not read the storage nodes from the smart contract?

Reply:
Just to clarify, with the changes in this PR, we're switching from

- read storage nodes from storage node registry at address `0xbAA81A0179015bE47Ad439566374F2Bae098686F`

to

- Override whatever is in the storage node registry with a list containing the node `0xde1112f631486CfC759A50196853011528bC5FA0`

This seems unintentional and unwanted - unless there is currently some problem with the previous approach. But I suspect the errors you were seeing were due to mixing up the tracker registry with the storage node registry, which was fixed on line 9.